gamemodes/terrortown/gamemode/shared/sh_equip_items.lua

```diff
@@ -439,15 +459,21 @@ if SERVER then
 
 		-- now set the individual random shop
 		if team then -- the shop is synced with the team
-			for _, ply in ipairs(plys and plys or player.GetAll()) do
+			local tmpTbl = plys and plys or player.GetAll()
```

Comment:
remove extra `plys and` check

Reply:
Also move this up a scope, since it is reused in line 473, all together with the for loop. So move the if inside a shared for loop.